packages/vite/src/client/hmrModuleRunner.ts

```diff
@@ -0,0 +1,83 @@
+import { createHotContext } from './client'
+
+declare const __FULL_BUNDLE_MODE__: boolean
+
+if (__FULL_BUNDLE_MODE__) {
+  class DevRuntime {
+    modules: Record<string, { exports: any }> = {}
+
+    static getInstance() {
+      // @ts-expect-error __rolldown_runtime__
+      let instance = globalThis.__rolldown_runtime__
+      if (!instance) {
+        instance = new DevRuntime()
+        // @ts-expect-error __rolldown_runtime__
+        globalThis.__rolldown_runtime__ = instance
+      }
+      return instance
+    }
+
+    createModuleHotContext(moduleId: string) {
+      return createHotContext(moduleId)
+    }
+
+    applyUpdates(_boundaries: string[]) {
+      //
+    }
+
+    registerModule(
+      id: string,
+      esmExportGettersOrCjsExports: Record<string, () => any>,
```

Comment:
```suggestion
      esmExportGettersOrCjsExports: Record<string, unknown>,
```
The type for the value of Record is `() => unknown` (esm) or `unknown` (cjs). So I think it's `unknown`.